bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/MetadataBookieDriver.java

```diff
@@ -68,6 +68,27 @@ LedgerManagerFactory getLedgerManagerFactory()
      */
     LayoutManager getLayoutManager();
 
+    /**
+     * Return health check is enable or disable.
+     *
+     * @return true if health check is enable, otherwise false.
+     */
+    default boolean isEnableHealthCheck() {
+        return true;
+    }
+
+    /**
+     * Disable health check.
+     */
+    default void disableHealthCheck(String enableHealthPath) {
```

Comment:
the enableHealthPath depends on the driver implementation
I believe that there is no need to pass the path here,
you can compute in the ZK driver

Reply:
also, this method would throw exceptions and also all the drivers are async.
I suggest to change the signature to
`CompletableFuture<Void> disableHealthCheck()`

here and for the other new methods

Reply:
OK, I will fixed